src/Jackett.Common/Definitions/hdbao.yml

```diff
@@ -0,0 +1,297 @@
+    download:
+      selector: a[href^="download.php?id="]
+      attribute: href
+    poster:
+      selector: img[data-src]
+      attribute: data-src
+    imdbid:
+      # site currently only has a badge and rating, the id is not present. just in case a future update.
+      selector: a[href*="imdb.com/title/tt"]
+      attribute: href
+    doubanid:
+      # site currently only has a badge and rating, the id is not present. just in case a future update.
+      selector: a[href*="movie.douban.com/subject/"]
+      attribute: href
+    date_elapsed:
+      # TODO: The selector from the example I copied includes
+      # `.rowfollow` which is a class for most of the column cells and
+      # AFAICT is only for presentation. As such I think it makes the
+      # selector more fragile and I've removed it from this
+      # definition. Should it be cleaned up in other definitions?
+      #
+      # time type: time elapsed (default)
+      selector: td:nth-child(4) > span[title]
+      attribute: title
+      optional: true
+      filters:
+        - name: append
+          # TODO: This is another demonstration of why a more verbose comment is
+          # valuable here. The `CST` abbreviation is used for both `-04:00` through
+          # `-06:00` in the Americas *and* `+08:00` in Asia. The more verbose comment
+          # removes ambiguity and saves contributor time.
+          args: " +08:00" # Asia/Taipei, CST
+        - name: dateparse
+          args: "yyyy-MM-dd HH:mm:ss zzz"
+    # TODO: I couldn't find an example of this when I sorted the full
+    # listing by date in both directions in the web UI of the exampale
+    # I copied. Where can I see the need for this in the example I
+    # copied so I can identify when it's needed for future
+    # definitions? Is it necessary for this definition?
+    date_added:
+      # time added
+      selector: td:nth-child(4):not(:has(span))
+      optional: true
+      filters:
+        - name: append
+          args: " +08:00" # CST
+        - name: dateparse
+          args: "yyyy-MM-ddHH:mm:ss zzz"
+    date:
+      text: "{{ if or .Result.date_elapsed .Result.date_added }}{{ or .Result.date_elapsed .Result.date_added }}{{ else }}now{{ end }}"
+    size:
+      selector: td:nth-child(5)
+    seeders:
+      selector: td:nth-child(6)
+    leechers:
+      selector: td:nth-child(7)
+    grabs:
+      selector: td:nth-child(8)
+    # TODO: There are no results for "2X", "2X 50%", and "30%" so I
+    # can't confirm their selectors but there are other examples in
+    # the example I copied of adding presumprive support for `imdbid`
+    # and `doubanid`, so I included them here.
+    #
+    # TODO: While testing, I could only find instances of "2X Free"
+    # and "ordinary" when I clicked through each of the top-level
+    # categories above, so those two are the only I could confirm by
+    # testing in the Prowlarr search UI. Is there a way I could have
+    # confirmed the rest?
```

Comment:
```suggestion
```

The template seems to be fairly consistent, so if you can see a couple being the same, it's safe enough to assume the rest will be too.